enarx-keep-sev-shim/asm/entry_ram64.s

```diff
@@ -0,0 +1,60 @@
+# SPDX-License-Identifier: Apache-2.0
+
+# This is the elf entry point called by enarx-keep-sev
+#
+# It sets up essential registers, page tables and jumps in kernel virtual address space
+# to the `_start_main` rust function.
+#
+# Arguments:
+# %rdi  = first parameter for start function
```

Comment:
Just for my clarification. The host-side VMM is supposed to set this up properly? Is that correct? So we're presuming that both the initial state of `%rdi` and the memory it points to are all measured. I think that's probably okay. I just want to make sure I understand the invariants. They should probably be documented here.

Reply:
yes